Applications/VGG/jni/main.cpp

```diff
@@ -380,61 +95,53 @@ int main(int argc, char *argv[]) {
   std::string config = args[0];
   resource = args[1];
 
-  srand(time(NULL));
-  std::vector<std::vector<float>> inputVector, outputVector;
-  std::vector<std::vector<float>> inputValVector, outputValVector;
-
-  std::vector<float> input_, label_;
+  std::array<UserDataType, 2> user_datas;
 
-  count_train.remain = num_class * num_train;
-  count_train.duplication.resize(count_train.remain);
-
-  count_val.remain = num_class * num_val;
-  count_val.duplication.resize(count_val.remain);
-
-  for (unsigned int i = 0; i < count_train.remain; ++i)
-    count_train.duplication[i] = i;
-
-  for (unsigned int i = 0; i < count_val.remain; ++i)
-    count_val.duplication[i] = i;
-
-  std::unique_ptr<ml::train::Dataset> db_train, db_valid;
   try {
-    db_train = ml::train::createDataset(ml::train::DatasetType::GENERATOR,
-                                        getBatch_train_file);
-    db_valid = ml::train::createDataset(ml::train::DatasetType::GENERATOR,
-                                        getBatch_val_file);
-  } catch (std::invalid_argument &e) {
-    std::cerr << "failed to createDataset, reason: " << e.what() << std::endl;
-    return 1;
+    user_datas = createRealDataGenerator(resource, batch_size, 1);
+  } catch (const std::exception &e) {
+    std::cerr << "uncaught error while creating data generator! details: "
+              << e.what() << std::endl;
+    return EXIT_FAILURE;
```

Comment:
In case of error, this application return 1 (EXIT_FAILURE) at lint 105.
but others in error case, it return 0.
How about unifying the return value as '1' or 'EXIT_FAILURE'?

Reply:
done. Thanks